src/notifications/notifications.service.ts

```diff
@@ -55,11 +51,11 @@ export class NotificationsService {
                 'GOV_NOTIFY_SAVED_SEARCH_NOTIFICATION_EMAIL_TEMPLATE_ID',
             );
         this.HOST = this.configService.get<string>('HOST');
-        this.FRONT_END_HOST = this.configService.get<string>('FRONT_END_HOST');
     }
 
     async processGrantUpdatedNotifications() {
         console.log('Running Process Grant Updated Notifications...');
+        console.log('Running Process Grant Updated Notifications.. OLD ONE.');
```

Comment:
Remove one of these logs / make it clearer plz

Reply:
yeah sorry this was just a leftover debug log, removed